newsletter-september-2022.rst

```diff
@@ -0,0 +1,68 @@
+and it looks like continued focus on polishing core platform features.
+We have a *lot* of features,
+and we need to continue making them easier to use.
+
+New features
+------------
+
+The latest updates from our team:
+
+- 🎉 We have soft launched the `initial version of our new landing pages <https://about.readthedocs.com/>`_ and started sharing it with the community. We plan to eventually have this as our primary landing page once we've gotten more feedback on it. 
+- We have improved out logic for `cancelling builds when a commit is pushed to an already building version <https://github.com/readthedocs/readthedocs.org/pull/9549>`_. This will improve user experience waiting on a build that will be overwritten, and save our servers some time as well!
+- We continued improving our docs pages. This month we did `badges <https://github.com/readthedocs/readthedocs.org/pull/9580>` & `build troubleshooting <https://docs.readthedocs.io/en/stable/build-troubleshooting.html>`_.
+- A lot of work has continued to standardize our codebase on the ``djstripe`` package, instead of our own custom stripe webhook implementation.
+- We improved our custom domain handling so that users who haven't properly activated their domain will get reminded about it a few times, and then we'll stop trying to configure it.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+Upcoming features
+-----------------
+
+- We continue to work on improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- We are working to redesign some of our search to make it nicer across our dashboard and in-doc search experiences. 
+- Our telemetry data is starting to really come together, and we're gathering insights on interesting metrics around builds. Expect a few blog posts about this in the coming months.
+- The 1.1 release of our `sphinx_rtd_theme` had a large amount of work done, and is nearing release.
+
+Possible issues
+---------------
+
+We have been having an ongoing issue with our Gitlab integration,
+where we can't refresh OAuth tokens for users.
+This has caused GitLab integration to randomly break after users enable it.
+We believe we have found a fix for this,
+which will be deployed soon.
+
+We are planning to be more active in deprecating old and outdated approaches to using our platform in Q4.
+We don't have anything firm to announce here yet,
+but we do plan to be more active in removing these features in the coming months.
+
+We continue to actively deprecate jQuery from our code, as well as `guide the Sphinx ecosystem <https://github.com/sphinx-doc/sphinx/issues/10608>`_ through the transition. 
+
+----
+
+Considering using Read the Docs for your next documentation project?
+Check out `our documentation <https://docs.readthedocs.io/>`_ to get started!
+
+Questions? Comments? Ideas for the next newsletter? `Contact us`_!
```

Comment:
I was thinking if it does make sense to "promote a feature" in this closing paragraph on each newsletter. Something small that fits in a tip paragraph. For example:

```rst
.. tip::

  Did you know that you can connect your project to Slack? Get notified instantly, 
  and don't miss a failing build again! Full documentation at https://docs.readthedocs.io/en/stable/build-notifications.html
```

What do you think?

Reply:
@humitos I think that's a great idea!

Reply:
Yes, that would work better! Actually, that is exactly what I wanted originally, but I didn't know how to suggest doing it 😅 Happy that you mentioned it!

Reply:
Done!